src/_includes/article.html

```diff
@@ -1,14 +1,15 @@
 {% if item.category %}
   {% if item.asset %}
     {% assign url = item.asset %}
-    {% unless url contains "https://" %}
-      {% assign url = "/assets/" | append: url %}
+    {% unless url contains 'https://' %}
+      {% assign url = '/assets/' | append: url %}
```

Comment:
The `liquid` formatter didn't change these quotes, so not sure how this got in there.